tests/test_wheel.py

```diff
@@ -83,6 +88,8 @@ def test_scipy_like(wheel_scipy_like):
         #'mypkg/submod/unknown_filetype.npq',
     }
     assert wheel_contents(artifact) == expecting
+    assert 'none' not in wheel_filename(artifact)
```

Comment:
Wouldn't in make more sense to use `artifact.parsed_filename.group('abi')` instead?

Reply:
I prefer this for both readability and robustness I think. There must not be a `none` anywhere, so as a test that is independent from the implementation this seem better to me.

Reply:
Yes, but you can check the exact value of the different groups. This is just a nitpick though.

Reply:
Let's do both then.